.bowerrc

```diff
@@ -1,3 +1,3 @@
 {
-    "directory": "hu_web/hu_browser/static/hu_browser/bower/"
+    "directory": "hu_web/common-static/bower/"
 }
```

Comment:
So bower is outside the namespace, but everything else would be inside the namespace? Hmmm